src/spatialNavigation.js

```diff
@@ -313,6 +315,9 @@ class SpatialNavigation {
       this.debug = debug;
 
       if (!this.nativeMode) {
+        if (Number.isInteger(throttle) && throttle > 0) {
+          this.throttle = throttle;
```

Comment:
Can you also add it in the constructor please? To have it also initialized with 0, and also to have it reset to 0 when the service is destroyed? The same as with other flags like nativeMode

Reply:
done :3